gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/importExport/ExportPanel.java

```diff
@@ -372,7 +372,7 @@ private File buildFile(final boolean isCustomExport) {
 						line.add(studentGradeInfo.getStudentEid());
 					}
 					if (!isCustomExport || this.includeStudentName) {
-						line.add(studentGradeInfo.getStudentLastName() + ", " + studentGradeInfo.getStudentFirstName());
+						line.add(FormatHelper.htmlUnescape(studentGradeInfo.getStudentLastName()) + ", " + FormatHelper.htmlUnescape(studentGradeInfo.getStudentFirstName()));
```

Comment:
This stuff's just going into an excel or csv download, right? So there are no security concerns about unescaping student names?

Reply:
Right. No security concern. Just needs to be CSV-escaped which OpenCSV handles for us

Reply:
Yeah, OpenCSV will escape the data.